tests/launch-mock-event-recorder-tests.sh

```diff
@@ -31,13 +31,13 @@ export DBUS_SESSION_BUS_PID
 trap finally EXIT
 
 # Start the mock service and add its methods.
-python -m dbusmock --system \
+python3 -m dbusmock --system \
```

Comment:
I guess not; if you do it your way, you are checking that whatever `python3` happens to be first in the path has the right modules, so here you should also use whatever `python3` happens to be first in the path.